docs/api-reference/geo-layers/tile-layer.md

```diff
@@ -138,10 +138,17 @@ if (signal.aborted) {
 
 The pixel dimension of the tiles, usually a power of 2.
 
-The tile size represents the target pixel width and height of each tile when rendered. Smaller tile sizes display the content at higher resolution, while the layer needs to load more tiles to fill the same viewport.
+For geospatial viewports, tile size represents the target pixel width and height of each tile when rendered. Smaller tile sizes display the content at higher resolution, while the layer needs to load more tiles to fill the same viewport.
+
+For non-geospatial viewports, the tile size should correspond to the true pixel size of the tiles.
 
 - Default: `512`
 
+##### `zoomOffset` (Number, optional)
+
+For non-geospatial viewports, this offset changes the zoom level at which the tiles are fetched.
+
+- Default: `0`
```

Comment:
Call out that this number needs to be an integer?

Reply:
Good call.